hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -278,25 +280,24 @@ private void initInvocationTarget() throws Exception {
         remote = !context.thisAddress.equals(invTarget);
     }
 
-    void notifyError(Object error) {
-        assert error != null;
+    void notifyError(Throwable cause) {
+        notifyError(new ErrorResponse(cause, op.getCallId(), op.isUrgent()));
```

Comment:
This seems redundant. `notifyError(ErrorResponse response)` doesn't care about `ErrorResponse` object, it just unwraps the cause. I think existing version is better, no need to wrap/unwrap.

Reply:
:+1: